spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../../lib/takeaway.rb'
+require_relative '../../lib/menu.rb'
+require 'twilio-ruby'
+require 'dotenv'
+
+describe "FEATURE SPEC" do
+  let(:takeaway) { Takeaway.new(Menu.new, Calculator.new, Twilio::REST::Client.new(Dotenv.load["ACCOUNT_SID"], Dotenv.load["AUTH_TOKEN"])) }
```

Comment:
Metrics/LineLength: Line is too long. [140/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)